AutoStack/MineShell.js

```diff
@@ -119,14 +111,24 @@ MineShell.prototype._exit = function () {
 };
 
 MineShell.prototype._uploadWorld = function () {
+
 	this._log("Uploading world...");
-	for (const folder of this._config.syncFolders) {
-		AWS.sync(SERVER_FOLDER_PATH + folder, this._worldUrl + folder);
+	stdin.write("say cloud save initiated, server saving halted\n");
```

Comment:
I think this might cause an issue when uploadWorld is caused after the server is shutdown for the final sync.  At that point, I think its stdin will be closed and so attempting to write to it will probably cause an error.  I think you could add a check to `this._child.stdin.writableEnded` to see whether its safe to send commands.

Reply:
Done! I agree.